frb/data/FRBs/FRB_positions_astrometry_final.txt

```diff
@@ -0,0 +1,153 @@
+180924 — originally published Bannister et al. 2019 but corrected in Macquart et al. 2020
```

Comment:
Add a preamble to this file describing what it is and
why it is here.

Reply:
Done